File: results/py/NON_REPEATING_ELEMENT_1.cpp
int f_filled ( int arr [ ] , int n ) {
  int mp [ n ] ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) mp [ arr [ i ] ] ++ ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) if ( mp [ arr [ i ] ] == 1 ) return arr [ i ] ;
  return - 1 ;
}

File: results/java_codex/NON_REPEATING_ELEMENT_1.java
int firstNonRepeating(int arr[], int n) {
  unordered_map<int, int> m;
  for (int i = 0; i < n; i++) {
    if (m.find(arr[i]) != m.end()) {
      m[arr[i]]++;
    } else {
      m[arr[i]] = 1;
    }
  }
  for (int i = 0; i < n; i++) {
    if (m[arr[i]] == 1) {
      return arr[i];
    }
  }
  return -1;
}